index.bs

```diff
@@ -692,14 +692,33 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. For each |authenticator| currently available on this platform, perform the following steps:
 
-    1. Let |credentialList| be a new [=list=].
-
-    1. If <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> [=list/is not empty=], execute a 
-        platform-specific procedure to determine which, if any, credentials in
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> are present on this |authenticator| by matching with 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/id}}</code> and 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/type}}</code>, and set |credentialList| to
-        this filtered list.
+    <span class="note">Note: the procedure to determine the [=authenticators=] present on a platform is 
+        platform-specific.</span>
+
+    1. Let |credentialList| be a new [=ordered set=].
+
+    1. If <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code>:
+        <dl class="switch">
+            : [=list/is not empty=]
+            :: [=set/For each=] credential |C| in <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code>:
+                1. If there is any credential |X| present on this |authenticator| that:
+                    - is mapped to |rpId|, and,
+                    - matches with <code>|C|.{{ScopedCredentialDescriptor/id}}</code> and 
+                        <code>|C|.{{ScopedCredentialDescriptor/type}}</code>,
+
+                    then [=set/append=] |X| to |credentialList|.
```

Comment:
Hm. `{{Transport}}` values are hints, and  `{{ScopedCredentialDescriptor}}.transports` is a `sequence<Transport>`, so we have so far assumed that an authenticator MAY be available over more than one transport, and that the client MAY select one such transport to use in an attempt to invoke `authenticatorGetAssertion` on |authenticator|. 

wrt..
> Semantically, a ScopedCredentialDescriptor with no transports is equivalent to one that lists all three...

..hm, I suppose so, modulo "using local configuration knowledge of the appropriate transport to use with |authenticator|"...